content/en/platform/corda/5.3/reference/corda-cli/cpi.md

```diff
@@ -0,0 +1,81 @@
+---
+description: "List of CPI commands for the Corda CLI. Use these commands to upload or fetch CPIs."
+date: '2024-04-24'
+menu:
+  corda53:
+    identifier: corda53-cordacli-cpi
+    weight: 500
+    parent: corda53-cli-reference
+title: "cpi"
+---
+# cpi
+
+This section lists the Corda CLI `cpi` arguments. You can use these commands to upload CPIs to your Corda 5 cluster as well as list the CPIs that are already there.
+
+## upload
+
+The `upload` command uploads CPIs present in the specified `.cpi` file into your Corda 5 cluster, so that virtual nodes can be created using this CPI at a later point.
```

Comment:
This reads a little funny to me. Can one `.cpi` file contains multiple CPIs?

Also, we usually just say "Corda" (not "Corda 5").

Reply:
@ShYogesh-R3 Can one .cpi file contains multiple CPIs?

Reply:
No, .cpi itself is a CPI file. it does not contain multiple CPIs. `.cpi` format represent the cpi file.

Reply:
Updated

Reply:
I think you could just say something like:

> The `upload` command uploads the specified CPI into your Corda cluster, so that virtual nodes can be created using this CPI at a later point.

Reply:
Yes, more concise. Updated.